concent_api/core/tasks.py

```diff
@@ -0,0 +1,40 @@
+import logging
+
+from celery import shared_task
+
+from conductor.models import VerificationRequest
+from verifier.tasks import blender_verification_order
+
+
+logger = logging.getLogger(__name__)
+
+
+@shared_task
+def upload_finished(subtask_id: str):  # pylint: disable=unused-argument
+    pass
+
+
+@shared_task
+def upload_acknowledged(subtask_id: str):
+    assert isinstance(subtask_id, str)
+
+    try:
+        verification_request = VerificationRequest.objects.get(subtask_id=subtask_id)
+    except VerificationRequest.DoesNotExist:
+        logging.error(
+            f'Task `upload_acknowledged` tried to get VerificationRequest object with ID {subtask_id} but it does not '
+            'exists.'
+        )
```

Comment:
Can we be a little less strict about line length? Splitting it like that really feels worse than just leaving it as:

``` python
except VerificationRequest.DoesNotExist:
    logging.error(f'Task `upload_acknowledged` tried to get VerificationRequest object with ID {subtask_id} but it does not exists.')
```

BTW: `does not exists` -> `does not exist`

Reply:
:+1: